doc/_extensions/zephyr/application.py

```diff
@@ -92,6 +92,9 @@ class ZephyrAppCommandsDirective(Directive):
     \:west-args:
```

Comment:
Typo in commit `doc: extend application sphinx extension to support flash arguments` text
s/flah-args/flash-args/

Reply:
fixed :)